src/ember-ops.ts

```diff
@@ -137,50 +138,95 @@ export class EmberOperation {
 }
 
 export function isEmberCliInstalled(): boolean {
+    let test = getEmberVersionDump();
+    return test ? true : false;
+}
+
+/**
+ * Returns ember -v console dump
+ * 
+ * @returns {string} 
+ */
+function getEmberVersionDump(): string {
     let emberBin = getPathToEmberBin();
 
     try {
         let exec = cp.execSync(`${emberBin} -v`, {
             cwd: getFullAppPath()
         });
 
+        let versionDump = exec.toString();
+
         console.log("Ember is apparently installed");
-        console.log(exec.toString());
+        console.log(versionDump);
 
-        return true;
+        return versionDump;
     } catch (e) {
         debugger;
 
-        return false;
+        return undefined;
+    }
+}
+
+var versionCache = false;
+/**
+ * Returns the versions from ember -v
+ * [0] = ember version
+ * [1] = node version
+ * [2] = os info
+ * 
+ * @returns {Array<string>} 
+ */
+function getVersionsFromDump(): Array<string> {
+    let matches;
+
+    if (!versionCache) {
+        let versionDump = getEmberVersionDump();
+        matches = versionDumpParse().exec(versionDump);
+        matches.shift();
+        versionCache = matches;
     }
+    else {
+        matches = versionCache;
+    }
+
+    return matches;
 }
 
 export function getEmberVersion(): Promise<string> {
     return new Promise((resolve, reject) => {
-        let bower;
+        let bower, bowerPath;
 
         if (!workspace || !workspace.rootPath) {
             return reject(new Error("Could not determine Ember version: Workspace not available."));
         }
 
         // Try go require the bower.json
-        try {
-            bower = require(path.join(getFullAppPath(), "bower.json"));
-        } catch (error) {
-            return reject(new Error("Could not determine Ember version: Bower.json not found."));
-        }
+        bowerPath = path.join(getFullAppPath(), "bower.json");
+        if (hasFile(bowerPath)) {
```

Comment:
I think if someone has Ember itself as a NPM package but a bower.json for other dependencies, the extension would assume that there is no Ember in the project.